akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -151,7 +152,7 @@ class CircuitBreaker(
       maxFailures: Int,
       callTimeout: FiniteDuration,
       resetTimeout: FiniteDuration) = {
-    this(scheduler, maxFailures, callTimeout, resetTimeout, 36500.days, 1.0)(executor)
+    this(scheduler, maxFailures, callTimeout, resetTimeout, 36500.days, 1.0, 0.0)(executor)
```

Comment:
(perhaps use a named parameter here (`, randomFactor = 0.0`) to make it easier to read)

Reply:
updated